CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 PUT_CHANGELOG_HERE
 
+- Deprecate `ApolloStore`/`NormalizedCache` `clearAll()` in favor of `removeAll()` for consistency with `remove()`
```

Comment:
Note that in Java and Kotlin collections, `removeAll()` takes a list of items to remove, while `clear()` clears the whole collection

Reply:
Fair! Oh well `clearAll()` isn't that bad after all.